eclass/acct-user.eclass

```diff
@@ -222,22 +222,25 @@ elockuser() {
 	eislocked "$1"
 	[[ $? -eq 0 ]] && return 0
 
+	local opts
+	[[ ${ROOT} == "/" ]] ||	opts=( -R "${ROOT}" )
+
 	case ${CHOST} in
 	*-freebsd*|*-dragonfly*)
-		pw lock "$1" || die "Locking account $1 failed"
-		pw user mod "$1" -e 1 || die "Expiring account $1 failed"
+		pw lock "${opts[@]}" "$1" || die "Locking account $1 failed"
```

Comment:
Have you verified (e.g. by reading manpages) that all these tools support `-R`?

Reply:
Yes, https://www.freebsd.org/cgi/man.cgi?pw(8)
First option in the description section.

But I did not test that, unfortunately :/

Reply:
FreeBSD seems to work fine but NetBSD doesn't have `-R` option. I guess you have to make non-/ ROOT support OS-conditional then.